### DISCUSSION TITLE
v1.3.21

### DISCUSSION THREAD

**Discussion by MartiUK** [cmderdev/cmder#2796] [General]
# IMPORTANT

**This release updates a dependency with a known vulnerability, it is suggested that you update as soon as possible.**

## What's Changed
* Added pagent ssh auth support by @AlphaGit in https://github.com/cmderdev/cmder/pull/1391
* Update issue template labels by @DRSDavidSoft in https://github.com/cmderdev/cmder/pull/2763
* Git-bash use conemu-msys2-64 connector by @nexplorer-3e in https://github.com/cmderdev/cmder/pull/2765
* Updates to `2` vendored dependencies by @github-actions in https://github.com/cmderdev/cmder/pull/2767
* Updates to `1` vendored dependencies by @github-actions in https://github.com/cmderdev/cmder/pull/2770
* Improvements to Cmder prompt by @DRSDavidSoft in https://github.com/cmderdev/cmder/pull/2772
* Updates to `1` vendored dependencies by @github-actions in https://github.com/cmderdev/cmder/pull/2773
* Updates to `1` vendored dependencies by @github-actions in https://github.com/cmderdev/cmder/pull/2776
* Improvements to shells and general fixes by @DRSDavidSoft in https://github.com/cmderdev/cmder/pull/2779
* Updates to `1` vendored dependencies by @github-actions in https://github.com/cmderdev/cmder/pull/2780
* Fix https://github.com/cmderdev/cmder/issues/2789 by @chrisant996 in https://github.com/cmderdev/cmder/pull/2791
* Updates to `3` vendored dependencies by @github-actions in https://github.com/cmderdev/cmder/pull/2786
* ⬆️ Update dependencies (clink v1.4.4) by @github-actions in https://github.com/cmderdev/cmder/pull/2792
* Update conemu to mitigate CVE-2022-46387 by @MartiUK in https://github.com/cmderdev/cmder/pull/2794

## New Contributors
* @AlphaGit made their first contribution in https://github.com/cmderdev/cmder/pull/1391
* @nexplorer-3e made their first contribution in https://github.com/cmderdev/cmder/pull/2765

**Full Changelog**: https://github.com/cmderdev/cmder/compare/v1.3.20...v1.3.21

<hr /><em>This discussion was created from the release <a href='https://github.com/cmderdev/cmder/releases/tag/v1.3.21'>v1.3.21</a>.</em>

**Comment by endolith** [comment]
It contains a file named `Version 1.3.20.151`, not `Version 1.3.21`
